src/main/java/com/google/netpcapanalysis/caching/policy/MaximumItemPolicy.java

```diff
@@ -0,0 +1,52 @@
+package com.google.netpcapanalysis.caching.policy;
+
+import com.google.netpcapanalysis.caching.DSCacheObject;
+import com.google.netpcapanalysis.interfaces.caching.Cache;
+import java.io.Serializable;
+
+public class MaximumItemPolicy<K, V extends Serializable> extends EvictionPolicy<K, V> {
+
+  private final long max;
+
+  public MaximumItemPolicy(long max) {
+    this.max = max;
+  }
+
+  @Override
+  public DSCacheObject<V> onGet(Cache<K, V> cache, DSCacheObject<V> data) {
+    return data;
+  }
+
+  @Override
+  public DSCacheObject<V> onPut(Cache<K, V> cache, DSCacheObject<V> data) {
+    return data;
+  }
+
+  @Override
+  public boolean checkCacheObjectEvict(Cache<K, V> cache, DSCacheObject<V> data) {
+    return true;
```

Comment:
Add a comment why you are returning true always since the expiration time is not used?

Reply:
Added